simulation_ws/src/turtlebot3_description_reduced_mesh/urdf/turtlebot3_waffle.urdf

```diff
@@ -1,12 +1,52 @@
 <?xml version="1.0" ?>
 <robot name="turtlebot3_waffle" xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:include filename="$(find turtlebot3_description_reduced_mesh)/urdf/common_properties.xacro"/>
```

Comment:
How come we're not including these anymore?

Reply:
In turtlebot3 (ROS2 branch as below link), They (turtlebot3 community) have merged these common_properties from common_properties.urdf into all three models urdf files (burger, waffle & waffle_pi) as below link.
https://github.com/ROBOTIS-GIT/turtlebot3/tree/ros2/turtlebot3_description/urdf
Hence We have followed the same.